homework/address.txt

```diff
@@ -1 +1 @@
-replace the whole line with your repo address - {user}/{repo}
+ kamilwojcik/git-checker 
```

Comment:
you need to provide a repo to so called "git-majonez" repository, not git-checker ;)